.gitignore

```diff
@@ -1,4 +1,5 @@
 # External dependencies
+.idea  # webstorm/intelliJ ide's etc
```

Comment:
Replace the `external dep` with `ide folders`

Reply:
sure, sorted 👍